imsv-docs-astro/src/content/docs/guides/regions.mdoc

```diff
@@ -53,3 +53,20 @@ Below is an indicative list of regions supported by Immersve.
 - United Arab Emirates
 - United Kingdom
 - United States
+
+## Set up environment variables
```

Comment:
There's missing context here. Should this game into the integrator guides?

Reply:
Hmm. yeah it could go into an integrator guide, or it could be on it's own and referenced by that guide. I'm not sure which way we want to go.

The one thing i noted was that since this was in the guides section it didn't seem to have the same level of guidance in it as our other guides, i.e. the script to quickly get going. So thought it would be good to add that since otherwise it just had a link to the api doc.

Reply:
i just re-ordered the content so that the script and the link to the guide is right after the description, and the indicative regions are after the script

Reply:
Talking to @catluin 

we should:
* update the guides/region page to include both custodial and non-custodial commands
* Update the web3 wallet guide to point to the region guide
* Add links to the integrator guides from the region guide

* @ncjones there currently isn't a [section about regions in the custodial guide](https://docs.immersve.com/guides/custodial-on-chain-card-issuing-integration/). Should there be?